cmd/interchaincmd/signatureaggregatorcmd/start.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package signatureaggregatorcmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ava-labs/avalanche-cli/cmd/flags"
+	"github.com/ava-labs/avalanche-cli/pkg/cobrautils"
+	"github.com/ava-labs/avalanche-cli/pkg/localnet"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/networkoptions"
+	"github.com/ava-labs/avalanche-cli/pkg/signatureaggregator"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+
+	"github.com/spf13/cobra"
+)
+
+var startNetworkOptions = []networkoptions.NetworkOption{
+	networkoptions.Local,
+	networkoptions.Fuji,
+}
+
+type StartFlags struct {
+	Network     networkoptions.NetworkFlags
+	SigAggFlags flags.SignatureAggregatorFlags
+	l1          string
+}
+
+var startFlags StartFlags
+
+// avalanche interchain signatureAggregator start
+func newStartCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "start",
+		Short: "starts signature aggregator",
+		Long:  `Starts locally run signature aggregator for the specified network with the L1's aggregator peers'`,
+		RunE:  start,
+		Args:  cobrautils.ExactArgs(0),
+	}
+	networkoptions.AddNetworkFlagsToCmd(cmd, &startFlags.Network, true, startNetworkOptions)
+	cmd.Flags().StringVar(&startFlags.l1, "l1", "", "name of L1")
+	return cmd
+}
+
+func start(_ *cobra.Command, _ []string) error {
+	network, err := networkoptions.GetNetworkFromCmdLineFlags(
+		app,
+		"",
+		startFlags.Network,
+		false,
+		false,
+		startNetworkOptions,
+		"",
+	)
+	if err != nil {
+		return err
+	}
+	if network.Kind == models.Local {
+		isRunning, err := localnet.IsLocalNetworkRunning(app)
+		if err != nil {
+			return err
+		}
+		if !isRunning {
+			return fmt.Errorf("unable to start local signature aggregator for local network as local network is not running. Run avalanche network start to start local networl")
+		}
+	}
+	if err = createLocalSignatureAggregator(network); err != nil {
```

Comment:
can you check if already started, if it is, tell the user on that and print information